curation-migration-datasync/eventConsumer.integration.ts

```diff
@@ -324,6 +334,253 @@ describe('event consumption integration test', function () {
       expect(dymamoDbSpy.callCount).toEqual(0);
     });
   });
+
+  describe('update-approved-item', () => {
```

Comment:
Given the number of fixtures, it would probably be worth it to start putting the test data elsewhere